File: Assets/Scripts/Fall.js
#pragma strict

function OnTriggerEnter(other : Collider) {
	if(other.gameObject.CompareTag("Player")) {
		SceneManagement.SceneManager.LoadScene(SceneManagement.SceneManager.GetActiveScene().name); // notre scène courante, càd "Scene2Interieur"
	} else {
		Destroy(other.gameObject);
	}
}

File: Assets/Scripts/Fall.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.SceneManagement;

public class Fall : MonoBehaviour {

	void OnTriggerEnter(Collider other) {
		if(other.gameObject.CompareTag("Player")) {
			SceneManager.LoadScene(SceneManager.GetActiveScene().name); // notre scène courante
		} else {
			Destroy(other.gameObject);
		}
	}
}